CMakeLists.txt

```diff
@@ -72,6 +72,11 @@ endif()
 if(BUILD_TIDY_TOOL)
   add_subdirectory(fairmq/tidy)
 endif()
+
+add_custom_target(codemeta
+  python3 ${CMAKE_SOURCE_DIR}/meta_update.py --outdir ${CMAKE_BINARY_DIR} --set-version ${PROJECT_VERSION}
```

Comment:
I would expect a `find_package(2)(Python … something interpreter)` for this, right?

Reply:
Ya, was considering it, but there was so much noise with CMake's Python detection in the past that I am afraid it will introduce more maintenance work than hardcoding it - this target really does not need to be super platform independent.

Reply:
Okay, got that! Please add a comment above it to that fact. Something like "meant for release management, not needed to be as portable".

Have you looked into `add_custom_command(OUTPUT … MAIN_DEPENDENCY …)` or the `SOURCES` part of `add_custom_target`?